Nef_3/doc/Nef_3/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h

```diff
@@ -5,9 +5,36 @@ namespace CGAL {
 /// Note that contrary to `Nef_polyhedron_3::convert_to_polyhedron()`, the output is not triangulated
 /// (but faces with more than one connected component of the boundary).
 /// The polygon mesh can be triangulated by setting `triangulate_all_faces` to `true` or by calling the function `triangulate_faces()`.
+/// @tparam Nef_polyhedron an object of type `Nef_polyhedron_3`.
+/// @tparam Polygon_mesh a model of `MutableFaceGraph`.
```

Comment:
_with an internal property map for `CGAL::vertex_point_t`._ Then you should adapt the sentence from [`copy_face_graph`](https://doc.cgal.org/latest/BGL/group__PkgBGLHelperFct.html#ga2d3eb044bfe42eab566779ce32216c11) about the Cartesian converter.